lms/static/sass/elements/_system-feedback.scss

```diff
@@ -145,9 +145,955 @@
+    right: $baseline;
+    padding: ($baseline/4) ($baseline/2) 0 ($baseline/2);
+    background: $gray-d4;
+    text-align: center;
+
+    .label {
+      @extend %cont-text-sr;
+    }
+
+    .icon {
+      @extend %t-icon6;
+      color: $white;
+      width: auto;
+      margin: 0;
+      padding: 2px;
+    }
+
+    &:hover {
+      background: $gray-d1;
+    }
+  }
+
+  // with dismiss (to sunset action-alert-clos)
+  .action-dismiss {
+
+    .button {
+      @extend %btn-secondary-white !optional;
+      padding:($baseline/4) ($baseline/2);
+    }
+
+    .icon,.button-copy {
+      display: inline-block;
+      vertical-align: middle;
+    }
+
+    .icon {
+      @extend %t-icon4;
+      margin-right: ($baseline/4);
+    }
+
+    .button-copy {
+      @extend %t-copy-sub1;
+    }
+  }
+}
+
+// ====================
+
+// js enabled
+.js {
+
+  // prompt set-up
+  .wrapper-prompt {
+    visibility: hidden;
+    pointer-events: none;
+
+    .prompt {
+
+    }
+  }
+
+  // prompt showing
+  &.prompt-is-shown {
+
+    .wrapper-prompt.is-shown {
+      visibility: visible;
+      pointer-events: auto;
+
+      .prompt {
+        @include animation(bounceIn $tmg-f1 ease-in-out 1);
+      }
+    }
+  }
+
+  // prompt hiding
+  &.prompt-is-hiding {
+
+    .wrapper-prompt {
+
+      .prompt {
+        @include animation(bounceOut $tmg-f1 ease-in-out 1);
+      }
+    }
+  }
+
+  // alert showing/hiding done by jQuery
+  .wrapper-alert { }
+
+  // notification showing/hiding
+  .wrapper-notification {
+    bottom: -($ui-notification-height);
+
+    // varying animations
+    &.is-shown {
+      @include animation(notificationSlideUp $tmg-s1 ease-in-out 1);
+      @include animation-fill-mode(forwards);
+    }
+
+    &.is-hiding {
+      @include animation(notificationSlideDown $tmg-s1 ease-in-out 1);
+      @include animation-fill-mode(forwards);
+    }
+  }
+}
+
+// ====================
+
+// block-level messages and validation
+.wrapper-message {
+
+  .message {
+    @extend %t-copy-sub1;
+    background-color: $gray-d2;
+    padding: ($baseline/2) ($baseline*0.75);
+    color: $white;
+
+    .icon {
+      font-style: normal;
+    }
+
+    &.information {
+      @extend %t-copy-sub1;
+      background-color: $gray-l5;
+      color: $gray-d2;
+    }
+
+    &.validation {
+      background-color: $gray-d2;
+      color: $white;
+
+      a {
+        color: $blue-l2;
+      }
+    }
+
+    &.has-warnings {
+      border-bottom: 3px solid $orange;
+
+      .fa-warning {
+        margin-right: ($baseline/2);
+        color: $orange;
+      }
+    }
+
+    &.has-errors {
+      border-bottom: 3px solid $red-l2;
+
+      .fa-exclamation-circle {
+        margin-right: ($baseline/2);
+        color: $red-l2;
+      }
+    }
+  }
+
+  .message-list {
+    margin-bottom: 0;
+  }
+
+  .message-actions {
+    padding: ($baseline/2) $baseline;
+    background-color: $gray-d1;
+
+    .actions-list {
+      @extend %actions-list !optional;
```

Comment:
@marcotuts made `!optional` otherwise need to copy dependancies from studio, also `btn-secondary-white` is not being used in dialog.